### ISSUE TITLE
Errors with recent merges to BottomBarPageRenderer

### ISSUE THREAD

**Issue by cpboyd** [thrive-now/BottomNavigationBarXF#47]
It seems like the merge https://github.com/thrive-now/BottomNavigationBarXF/commit/d033551cc1506ed6846127ef4933ab1e0a69be65 kind of jumbled some stuff up.

[Line 46](https://github.com/thrive-now/BottomNavigationBarXF/blob/master/BottomBar.Droid/Renderers/BottomBarPageRenderer.cs#L46):  Missing the declaration `UpdatableBottomBarTab[] _currentTabs;`

[Line 307](https://github.com/thrive-now/BottomNavigationBarXF/blob/master/BottomBar.Droid/Renderers/BottomBarPageRenderer.cs#L307): Looking through the commits, it seems like there should be `_currentTabs = tabs;`.  However, the type of `tabs` would also need to change from `BottomBarTab[]` to `UpdatableBottomBarTab[]` with [line 301](https://github.com/thrive-now/BottomNavigationBarXF/blob/master/BottomBar.Droid/Renderers/BottomBarPageRenderer.cs#L46) replaced with
```c#
return new UpdatableBottomBarTab(tabIconId, page.Title, page.Id);
```

However, this requires the definition of the class `UpdatableBottomBarTab`:
```c#
        private class UpdatableBottomBarTab : BottomBarTab
        {
            public Guid PageId { get; private set; }

            public UpdatableBottomBarTab(int iconResource, string title, Guid pageId) : base(iconResource, title)
            {
                PageId = pageId;
            }

            public void SetIconResource(int iconResource)
            {
                _iconResource = iconResource;
            }

            public int GetIconResource()
            {
                return _iconResource;
            }
        }
```

[Line 309](https://github.com/thrive-now/BottomNavigationBarXF/blob/master/BottomBar.Droid/Renderers/BottomBarPageRenderer.cs#L309): It's missing the function declaration:
```c#
		void SetTabColors()
		{
```

[Line 322](https://github.com/thrive-now/BottomNavigationBarXF/blob/master/BottomBar.Droid/Renderers/BottomBarPageRenderer.cs#L322): `tabColor.Value` should just be `tabColor` because it's no longer nullable.

**Comment by lazmeister** [comment]
Yes I just started reviewing it again with the Thrive team and it did indeed break many points.

**Comment by lazmeister** [comment]
we had to revert the three commits for the time being #50 #49 #48

**Comment by Im-PJ** [comment]
when it will be fixed. my project is not working after updating this library :(
